styleguide/derek/pattern-components/leads/lead-image/lead-image.ejs

```diff
@@ -0,0 +1,18 @@
+<div class="container standard-padding-top">
+  <div class="row flex-row">
+    <div class="col-md-9 flex-all">
+      <div class="subpanel">
+        <div class="subpanel-inner">
+          <%- partial("../lead/lead.ejs") %>
+        </div>
+      </div>
+    </div>
+    <div class="col-md-3 flex-all">
+      <div class="subpanel">
+        <div class="subpanel-inner">
+          <%- partial("../../images/image.ejs") %>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
```

Comment:
Added because it was needed and I forgot to during the hackathon. Same structure as lead-video.